src/components/ListDetails/ListDetails.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import * as S from './Style'
+import { serviceNode } from '../../services/node/node'
+
+import ItemDetails from '../ItemDetails/ItemDetails'
+import Loader from '../Loader/Loader'
+import Status from '../Status/Status'
+
+const ListDetails = ({ }) => {
+  const [expenseList, setExpenseList] = useState(undefined)
+  
+
+  useEffect(() => {
+    serviceNode.getExpenses().then((response) => response.json())
+    .then((data) => {
+      setExpenseList(data)
+    }).catch(error => {
+      setExpenseList(false)
+    })
+  }, [])
+
+  return(
+    <>
+      <S.Wrapper>
+        <S.Headline>List of Expense</S.Headline>
+        { expenseList === undefined ? <Loader /> : 
```

Comment:
Moze lepiej przeniesc to do jakiejsc metodki, zagniezdzanie ternary jest malo czytelne moim zdaniem

Reply:
można

Reply:
oprócz tego wystarczy `expenseList || <Loader />`

Reply:
juz nie, bo na start bedzie tablica